client/landing/stepper/declarative-flow/internals/steps-repository/processing-step/hooks/use-processing-loading-messages.ts

```diff
@@ -20,6 +22,34 @@ export function useProcessingLoadingMessages( flow?: string | null ) {
 		];
 	}
 
+	if ( isWooExpressFlow( flow || null ) ) {
+		return stepData.currentStep === 'siteCreationStep'
+			? [
+					{
+						title: __( 'Woo! We’re creating your store' ),
+						subtitle: __(
+							'#FunWooFact: Did you know that Woo powers almost 4 million stores worldwide? You’re in good company.'
+						),
+						duration: 1000,
+					},
+			  ]
+			: [
+					{
+						title: __( 'Applying the finishing touches' ),
+						subtitle: __(
+							'#FunWooFact: There are more than 150 WooCommerce meetups held all over the world! A great way to meet fellow store owners.'
+						),
+						duration: 6000,
+					},
+					{
+						title: __( 'Turning on the lights' ),
+						subtitle: __( '#FunWooFact: Our favorite color is purple 💜' ),
+						// Set a very long duration to make sure it shows until the step is completed
+						duration: 60000,
+					},
+			  ];
```

Comment:
We're seeing situations where some of the steps are taking a long time. While we have that slowness in the flow, I think we should be defensive about the timings here. It may even be worth showing the `Applying the finishing touches` message for longer.
```suggestion
			? [
					{
						title: __( 'Woo! We’re creating your store' ),
						subtitle: __(
							'#FunWooFact: Did you know that Woo powers almost 4 million stores worldwide? You’re in good company.'
						),
						duration: 2000,
					},
			  ]
			: [
					{
						title: __( 'Applying the finishing touches' ),
						subtitle: __(
							'#FunWooFact: There are more than 150 WooCommerce meetups held all over the world! A great way to meet fellow store owners.'
						),
						duration: 10000,
					},
					{
						title: __( 'Turning on the lights' ),
						subtitle: __( '#FunWooFact: Our favorite color is purple 💜' ),
						// Set a very long duration to make sure it shows until the step is completed
						duration: 150000,
					},
			  ];
```